manifests/config.pp

```diff
@@ -80,6 +80,35 @@
     ensure  => absent,
   }
 
+  if $::foreman::jobs_sidekiq {
+    if $::foreman::jobs_sidekiq_redis_url == undef {
+      class { '::redis':
+        default_install => false,
```

Comment:
I would rather keep this var passing if possible - we might need to include the class with default true in redis cache.

Reply:
So that's the part you can't do. Classes are singleton. That's why I'd prefer to keep it just to `include redis` and keep the control of it out of this module.

Reply:
Ok, including now instead.